### ISSUE TITLE
tf.layers.batch_normalization always occur error

### ISSUE THREAD

**Issue by xjcvip007** [Microsoft/MMdnn#586]
Platform (like ubuntu 16.04/win10):

Python version:  Anaconda 3.6.4

Source framework with version (like Tensorflow 1.4.1 with GPU): tf 1.11.0

Destination framework with version (like CNTK 2.3 with GPU): ONNX

Pre-trained model path (webpath or webdisk path): model.pb

Running scripts:mmconvert -sf tensorflow -iw face_model/model.pb --inNodeName inputs/input_img --inputShape 36,60,3 --dstNodeName dense_2/BiasAdd  -df onnx -om model.onnx

When I run the scripts, I always obtain the AssertionError:
name: "batch_normalization/cond/FusedBatchNorm/Switch_1:1"
op: "NoOp"
However, in the example, they use normalizer_fn=slim.batch_norm for the mobilenet_v2, it seem can be run correctly. What's the difference betwwen slim.batch_norm with tf.layers.batch_normalization for the MMdnn? Thx~

**Comment by rainLiuplus** [comment]
Hi @xjcvip007 , please use the latest code to have a try. Thanks!
